src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java

```diff
@@ -25,7 +25,7 @@ public class Anchor extends Module {
         .name("max-height")
         .description("The maximum height Anchor will work at.")
         .defaultValue(10)
-        .range(0, 255)
+        .range(0, 383)
```

Comment:
im pretty sure this is wrong.

1.18 coordinates are from `-64` to `320`

Reply:
pre 1.18 was up to 256, so its the total height limit -1 so I just did the new height -1
320 + 64 = 384, 384 - 1 = 383

Reply:
> im pretty sure this is wrong.
> 
> 
> 
> 1.18 coordinates are from `-64` to `320`

-64 to 320 and therefore its possible to be 383 (well more obviously) blocks above the bottom of the world

Reply:
as in it is relative to the floor beneath you (iirc)

Reply:
exactly